src/factories/pid_lut/PhaseSpacePID_factory.h

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2025, Simon Gardner
+
+#pragma once
+
+#include <edm4eic/MCRecoParticleAssociationCollection.h>
+#include <edm4eic/ReconstructedParticleCollection.h>
+#include <edm4hep/MCParticleCollection.h>
+#include <edm4hep/ParticleIDCollection.h>
+
+#include "algorithms/pid_lut/PhaseSpacePID.h"
+#include "extensions/jana/JOmniFactory.h"
+#include "services/algorithms_init/AlgorithmsInit_service.h"
+
+namespace eicrecon {
+
+class PhaseSpacePID_factory : public JOmniFactory<PhaseSpacePID_factory, PhaseSpacePIDConfig> {
+public:
+  using AlgoT = eicrecon::PhaseSpacePID;
+
+private:
+  std::unique_ptr<AlgoT> m_algo;
+
+  PodioInput<edm4eic::ReconstructedParticle> m_recoparticles_input{this};
+  PodioInput<edm4eic::MCRecoParticleAssociation> m_recoparticle_assocs_input{this};
+  PodioOutput<edm4eic::ReconstructedParticle> m_recoparticles_output{this};
+  PodioOutput<edm4eic::MCRecoParticleAssociation> m_recoparticle_assocs_output{this};
+  PodioOutput<edm4hep::ParticleID> m_particleids_output{this};
+
+  ParameterRef<std::string> m_system{this, "system", config().system, "For the ParticleID record"};
+  ParameterRef<std::vector<float>> m_direction{
+      this, "direction", config().direction,
+      "Direction vector for the phase space (default is along z-axis)"};
+  ParameterRef<double> m_opening_angle{this, "opening_angle", config().opening_angle,
```

Comment:
```suggestion
  ParameterRef<double> m_opening_angle{this, "openingAngle", config().opening_angle,
```
IIRC the convention is a camelCase for the name of the parameters.